redisinsight/ui/src/components/navigation-menu/NavigationMenu.tsx

```diff
@@ -315,33 +315,47 @@ const NavigationMenu = () => {
         </FeatureFlagComponent>
         <HelpMenu />
         {publicRoutes.map(renderPublicNavItem)}
-        <Divider colorVariable="separatorNavigationColor" className="eui-hideFor--xs eui-hideFor--s" variant="middle" />
-        <Divider
-          colorVariable="separatorNavigationColor"
-          className="eui-showFor--xs--flex eui-showFor--s--flex"
-          variant="middle"
-          orientation="vertical"
-        />
-        <EuiToolTip
-          content="Redis Insight Repository"
-          position="right"
+        <FeatureFlagComponent
+          name={FeatureFlags.disabledByEnv}
+          enabledByDefault
+          otherwise={(
+            <Divider
+              color="transparent"
+              className="eui-hideFor--xs eui-hideFor--s"
+              variant="middle"
+            />
```

Comment:
Do we need this transparent Divider?

Reply:
Kind of, if we don't have a divider at all, the sidebar jumps when you hover over the Help button, shows vertical and horizontal scrollbars.
If it is not transparent, then we get a divider that has nothing after it.
Maybe more elegant solution is possible but have to dig into how to disable the hover effect for the help button